rpa-builder/6.1/modules/ROOT/pages/toolbox-web-automation-web-session-chrome.adoc

```diff
@@ -27,26 +28,33 @@ we use.
 
 == Inbound Variables
 
-* Additional Chrome arguments
-* Alternative User Agent String
-* Web Session ID
-* Initial URL
+* *Additional Chrome arguments*
+* *Alternative User Agent String*
+* *Web Session ID*
+* *Initial URL*
 
 == Outbound Variables
 
 This action step has no outbound variables.
 
 // Wizard
-include::partial$toolbox-web-automation-web-session.adoc[tag=wizardCommon]
+include::partial$toolbox-web-automation-web-session.adoc[tags=wizardCommon;!pageLoadStrategy]
+
+=== Browser specific settings
 
-*Browser specific settings*:
+image:toolbox-variable-web-automation-web-session-images/image5.png[50%, 50%, An image showing the browser specific settings]

```

Comment:
Though it is an alt text, it would be great to add the full stop at the end of the sentence to improve the pronunciation of the speech machine.